lib/game.rb

```diff
@@ -3,23 +3,40 @@ class Game
 
   def initialize
     @board = Board.new
-    @players = []
+    @players = create_players
+    @message_bot = Message.new
   end
 
   def take_turn
```

Comment:
We'll probably need to break this up into different methods to focus on single responsibility of just taking a turn.